standard_autoencoder.py

```diff
@@ -60,7 +60,45 @@ def init_network(self):
                 self.initialization_function([input_dim, output_dim]))
             self.biases['decoder_b%i' % decoder_layer_idx] = tf.Variable(
                 self.initialization_function([output_dim]))
-        
+
+    def get_setter_ops(self):
+        self.weights_placeholders = {}
```

Comment:
can we comment what these variables do?

Reply:
/ what the overall logic is here.

Reply:
I think this looks good. Did you verify that this actually sets the values correctly? The assign op is weird to me, although it does look like it's being used correctly.